### ISSUE TITLE
sdwan_attach_feature_device_template device-specific variable keys

### ISSUE THREAD

**Issue by andreasbaekdahl** [CiscoDevNet/terraform-provider-sdwan#103]
When using the sdwan_attach_feature_device_template, it is unclear how the device-specific variables are supposed to be assigned to the device.

In simple cases we can use the variable keys specified in the feature template. Example:
site_id_variable = "var_site_id" in an sdwan_cisco_system_feature_template, means you have have to assign the following in sdwan_attach_feature_device_template per device:
```
variables {
  var_site_id = 100
}
```

In more complex cases where the same feature templates is assigned to device template multiple times, the API indicates through the /template/device/config/input endpoint that we need to use full paths of the variables. Example: the same BGP feature templates is assigned to two different VPNs: VPN 1 and VPN 2.
The /template/device/config/input API returns the following structure:
```
{
    ...
    "data": [
        {
            ...
            "/2//router/bgp/neighbor/bgp_neighbor_address/address": "",
            "/2//router/bgp/neighbor/bgp_neighbor_address/description": "",
            "/2//router/bgp/neighbor/bgp_neighbor_address/remote-as": "",
            "/1//router/bgp/neighbor/bgp_neighbor_address/address": "",
            "/1//router/bgp/neighbor/bgp_neighbor_address/description": "",
            "/1//router/bgp/neighbor/bgp_neighbor_address/remote-as": "",
        }
    ]
}
```

Can we expect to use those fully qualified keys directly with the provider?
It is not possible to assign these device-specific variables in the following way:
```
resource "sdwan_attach_feature_device_template" "example" {
  id       = sdwan_feature_device_template.example.id
  devices = [
    {
      id = var.device_id
      variables = {
        ...
        "/2//router/bgp/neighbor/bgp_neighbor_address/address" : "2.2.2.2"
        "/2//router/bgp/neighbor/bgp_neighbor_address/description" : "VPN2 neighbor"
        "/2//router/bgp/neighbor/bgp_neighbor_address/remote-as" : "2"
        "/1//router/bgp/neighbor/bgp_neighbor_address/address" : "1.1.1.1"
        "/1//router/bgp/neighbor/bgp_neighbor_address/description" : "VPN1 neighbor"
        "/1//router/bgp/neighbor/bgp_neighbor_address/remote-as" : "1"
      }
    }
  ]
}
```

Resulting error:
```
sdwan_attach_feature_device_template.example: Creating...
╷
│ Error: Client Error
│ 
│   with sdwan_attach_feature_device_template.example,
│   on test.tf line 226, in resource "sdwan_attach_feature_device_template" "example":
│  226: resource "sdwan_attach_feature_device_template" "example" {
│ 
│ Failed to attach device template, got error: Action push_feature_template_configuration-f51fe9a7-b768-4b4a-8340-e4e4d52d55f7 for device
│ C8K-2F579999-7AE8-FCCE-00FF-C1360C885E6E failed. Activity log: ["[20-Nov-2023 10:07:36 UTC] Configuring device with feature template: TF-Example","[20-Nov-2023
│ 10:07:37 UTC] Failed to update configuration - Error  invalid value for: address in /templates/template/vpn/vpn-instance/router/bgp/neighbor/address\n"]
```

**Comment by danischm** [comment]
https://github.com/CiscoDevNet/terraform-provider-sdwan/commit/4ca854109b2d824357e6cd733c657b86db33fa7d

**Comment by danischm** [comment]
Fixed in v0.2.11 release.
